homeassistant/components/comfoconnect/sensor.py

```diff
@@ -72,163 +76,187 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_LABEL = "label"
-ATTR_MULTIPLIER = "multiplier"
-ATTR_UNIT = "unit"
+
+@dataclass
+class ComfoconnectRequiredKeysMixin:
```

Comment:
Why make this a mix in and not just add to `ComfoconnectSensorEntityDescription` ?

Reply:
It's a workaround to add required keys without any default values. Necessary as dataclasses don't support reordering of fields.

Reply:
For reference: https://github.com/home-assistant/core/pull/53610